### ISSUE TITLE
Cache-Control: no-store only while upload is incomplete

### ISSUE THREAD

**Issue by ronag** [tus/tus-resumable-upload-protocol#106]
Once upload is complete caching should be allowed.

**Comment by Acconut** [comment]
Please be aware that the specification does only required the `Cache-Control: no-store` header for HEAD responses as the GET request/response is not defined by the protocol.
Even after an upload is finished, the HEAD response should not be cached as certain properties of the upload still may change, e.g. its expiration date or it may become totally unavailable. Therefore, I think, the HEAD response should not be cached.

**Comment by ronag** [comment]
>  as certain properties of the upload still may change

That is true. However, I think this should be a SHOULD and not a MUST once the upload has completed. The server might be able guarantee that it will not change if e.g. there is no expiration date.

**Comment by Acconut** [comment]
Agreed, closing in favor of #121.
